examples/custom-metrics/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use autometrics::autometrics;
+
+/// Example HTTP handler function
+#[autometrics]
+pub fn get_index_handler() -> Result<String, ()> {
+    Ok("Hello world!".to_string())
+}
+
+// Run the example with `--features=metrics` to use the `metrics` crate to define additional metrics.
+#[cfg(feature = "metrics")]
+pub fn function_with_custom_metrics_metric() {
```

Comment:
@axiomatic-aardvark added this example that shows how you can collect other metrics alongside the autometrics ones. Let me know what you think!